qcodes/instrument_drivers/Spectrum/M4i.py

```diff
@@ -11,15 +11,23 @@
 #
 
 #%%
+import os
+import sys
 import logging
 import numpy as np
 import ctypes as ct
 from functools import partial
 from qcodes.utils.validators import Enum, Numbers, Anything
 from qcodes.instrument.base import Instrument
+
+
 try:
+    # add the location of the pyspcm header file manually
+    header_dir = os.path.split(__file__)[0]
+    sys.path.append(header_dir)

```

Comment:
I think it's good practice to only append it if it's not already there.

Reply:
Ok, done